Makefile.global.in

```diff
@@ -17,6 +17,12 @@ postgres_abs_builddir:=@POSTGRES_BUILDDIR@
 PG_CONFIG:=@PG_CONFIG@
 PGXS:=$(shell $(PG_CONFIG) --pgxs)
 
+ifeq ($(enable_coverage),yes)
+	export PG_CPPFLAGS += -ftest-coverage -fprofile-arcs
```

Comment:
What's the difference between this and the `--coverage` flag used by `pg_shard` or `cstore_fdw`?